### ISSUE TITLE
publishers names with just numbers causes blank screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maloep/romcollectionbrowser#160]
```
I have found that publishers with just numbers causes the database not to load. 
Example syphron filter, publisher is 989, after loading need to clean database 
to show games, but still cannot use the filters properly at the top, once you 
hit the name/year etc with that publisher in it the filters freeze and dont 
change. 

Solution so far is to edit the nfo and remove the publisher (or in my case 
rename nineeightnine) and rescrape. Once that is done everything works as 
normal again.

```

Original issue reported on code.google.com by `pudd...@gmail.com` on 23 Jan 2014 at 5:39

**Comment by bruny** [comment]
Could not reproduce with RCB 2.1.2 on Kodi Isengard (15.2).

One thing to note is the SQL_CREATE.txt script creates the publisher table with [name] column set to datatype NONE. This needs to be changed to TEXT?

**Comment by bruny** [comment]
Fixed by PR #224
